src/seismometer/controls/thresholds.py

```diff
@@ -0,0 +1,148 @@
+        values = tuple(0 for _ in names) if not value else tuple(value)  # set initial values
+        values = tuple(min(1.0, max(0, val)) for val in values)  # clamp values to [0, 1]
+        self.value = {k: v for k, v in zip(self.names, values)}
+        self.sliders = {}
+        for name, val in self.value.items():
+            sub_slider = FloatSlider(
+                value=val,
+                min=0,
+                max=1.0,
+                step=0.01,
+                description=name,
+                disabled=False,
+                continuous_update=False,
+                orientation="horizontal",
+                readout=True,
+                readout_format=".2f",
+                style=WIDE_LABEL_STYLE,
+            )
+            self.sliders[name] = sub_slider
+            sub_slider.observe(self._on_slider_change, "value")
+
+        self.value_update_in_progress = False
+
+        super().__init__(children=list(self.sliders.values()), layout=Layout(width="max-content", min_width="300px"))
+        self.observe(self._on_value_change, "value")
+        self._disabled = False
+
+    @property
+    def disabled(self) -> bool:
+        return self._disabled
+
+    @disabled.setter
+    def disabled(self, disabled: bool):
+        self._disabled = disabled
+        for slider in self.sliders.values():
+            slider.disabled = disabled
+
+    def _on_slider_change(self, change=None):
+        """Slider has changed, update the value tuple"""
+        if self.value_update_in_progress:
+            return
+        if change:
+            if (slider := change["owner"]) in self.sliders.values():
+                self.value[slider.description] = slider.value
+
+    def _on_value_change(self, change=None):
+        """Bubble up changes to sliders"""
+        self.value_update_in_progress = True
+        if change:
+            if val := change["new"]:
+                for key, value in val.items():
+                    self.sliders[key].value = value
+        self.value_update_in_progress = False
+
+
+class MonotonicPercentSliderListWidget(PercentSliderListWidget):
+    """
+    Vertical list of sliders, with increasing values
+    """
+
+    def __init__(self, names: tuple[str], value: tuple[int] = None, increasing: bool = True):
+        """A vertical list of sliders
+
+        Parameters
+        ----------
+        names : tuple[str]
+            Slider names
+        value : Optional[tuple[str]], optional
+            Slider start values, by default None, starts all sliders at zero.
+        increasing : bool, optional = True
+            Forces sliders to be increasing, else decreasing.
+            If initial values are not sorted, raise an ValueError.
+        """
+        super().__init__(names, value)
+
+        if tuple(sorted(self.value.values(), reverse=not increasing)) != tuple(self.value.values()):
+            raise ValueError("Initial values are not sorted")
```

Comment:
Can you add expected direction to the error message?  They might be "sorted" but just need the increasing flag toggled.

Nit. pandas uses the flag ascending rather than increasing, should we align? (numpy doesn't seem to even expose the option)

Reply:
moving to ascending to match pandas (since python's reverse feels less intuitive)

FINC